### DISCUSSION TITLE
Wacom Drawing Tablet support

### DISCUSSION THREAD

**Discussion by ruchit-bhatt** [rustdesk/rustdesk#6065] [Feature Request]
Can i get more info on how to passthrough Wacom drawing tablet from remote to server ?

**Comment by ruchit-bhatt** [comment]
Need proper solution to passthrough drawing tablet and other USB devices.

**Reply by shamirahim111** [reply]
@ruchit-bhatt isnt this same as https://github.com/rustdesk/rustdesk/discussions/6014 ? if you solve usb passthrough in general, that should help with everything

**Reply by ruchit-bhatt** [reply]
@shamirahim111  
Yes, you're right. This is very important feature for media and entertainment industry. Thanks

**Reply by shamirahim111** [reply]
@ruchit-bhatt yes. we should request the team on #6014  to implement overall usb passthrough. the more people show interest in a particular feature, the more likely it is going to be implemented by the team. that is the hope anyway

**Reply by ruchit-bhatt** [reply]
I hope USB pass-through feature will support pressure sensitivity of drawing tablet.

**Comment by ruchit-bhatt** [comment]
Any update on this? Very important feature for artists, put this feature on top priority.

**Reply by amphyvi** [reply]
Wondering this as well

**Comment by Aleks048** [comment]
Hi,
is there any update on tablet support for rustdesk?

**Comment by open-trade** [comment]
Stylus support already added in 1.4.1, but I do not know if it is related to wacom, is it not stylus pen?

https://github.com/rustdesk/rustdesk/pull/12196

**Reply by k3-cat** [reply]
I think they refer to pressure and tilt supports (like what windows ink does).
Currently, 1.4.2, rustdesk only accept pen as a regular pointer input.

**Reply by LarsCawley** [reply]
I came here to check if pressure sense is supported. Here's hoping it can make it into the roadmap.
